bin/edge-sync

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+php /usr/src/code/app/cli.php edge-sync $@
```

Comment:
What is this task for? let's avoid the term edge as it doesn't 100% fit

Reply:
This task loops over payloads to delete the ones that were sent successfully and re-queue the failed ones.
Renamed to region